Sources/Apollo/GraphQLHTTPResponseError.swift

```diff
@@ -42,7 +42,7 @@ public struct GraphQLHTTPResponseError: Error, LocalizedError {
     guard
       let data = self.body,
       let json = try? self.serializationFormat.deserialize(data: data) as? JSONObject,
-      let errorArray = json["errors"] as? [JSONObject] else {
+      let errorArray = json?["errors"] as? [JSONObject] else {
```

Comment:
This shouldn't be necessary - we've unwrapped `json` directly above.

What version of Swift are you using?

Reply:
I'm just adding the code via cocoapods, and building it with XCode 11.2.1

Reply:
Using swift '5' 
5.1.2
I think 'some' swift variants are casting the 'try?' differently now.   Pushing another 'attempt'.

Reply:
Right - what version of Swift is your project using though, Swift 4 or Swift 5? If you're on 4, `try?` and `as?` on the same line can create a double-optional, whereas on 5, it should only create a single optional. This is related to the version of Swift you're using rather than the version of Xcode. 

Please note that this project supports Swift 5 and higher.

Reply:
oops...  I was setting it to swift 4.2...

Reply:
yeah.. this is my bad.   There was a bad post_install cocoa pod script that clobbered this to swift 4.2  
Not sure if you even want to support 4.2, so take the code or not...